.github/CODEOWNERS

```diff
@@ -258,6 +258,9 @@
 /modules/programs/waybar.nix                          @berbiche
 /tests/modules/programs/waybar                        @berbiche
 
+/modules/programs/wezterm.nix                         @blmhemu
+/tests/modules/programs/wezterm                       @blmhemu
```

Comment:
As GitHub displays in the warning: This entry does nothing.

Reply:
Hey !  Could you pls point me how to solve this. It seems some other entries are being flagged as well.

Reply:
What you have is fine. It won't automatically notify you or request you to review, though since you aren't a member of the repository.

Eventually we should probably figure out a better system, but for now it's fine.